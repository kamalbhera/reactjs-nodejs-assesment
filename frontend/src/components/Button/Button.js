import React from 'react';

import './Button.scss';

const Button = () => {
	return (
		<div className='btn-primary-apply'>
			<button>Apply</button>
		</div>
	);
};

export default Button;
