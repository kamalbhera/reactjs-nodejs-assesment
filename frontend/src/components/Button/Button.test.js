import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';

it('renders Button component', () => {
	const result = shallow(<Button />);
	const html = <div className="btn-primary-apply"><button>Apply</button></div>;
	expect(result).toMatchElement(html);
	expect(result).toMatchSnapshot();
});
