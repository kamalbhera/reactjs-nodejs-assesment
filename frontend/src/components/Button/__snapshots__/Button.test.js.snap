// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button component 1`] = `
<div
  className="btn-primary-apply"
>
  <button>
    Apply
  </button>
</div>
`;
